include/sys/zil_impl.h

```diff
@@ -222,6 +222,9 @@ struct zilog {
 	 * (see zil_max_copied_data()).
 	 */
 	uint64_t	zl_max_block_size;
+
+	/* Pointer for per dataset zil sums */
+	zil_sums_t *zil_sums;
```

Comment:
`zl_sums` in keeping with the naming convention.

Reply:
Updated